MEs_Proposal_Guide.md

```diff
@@ -0,0 +1,101 @@
+
+- Example: ⚪ "Fight or Flight" will have you fight a boss PKMN from the biome, for a somewhat good reward. You will rarely win or lose your run here.
+- Example: 🔴 "Dark Deal" will remove a Pokémon from your party. This can be your best Pokémon at the time. You **__can catch a legendary__** which is good for the run, and your box, but you also very well **__can lose the run and not catch anything__**
+
+### Requirements (Somewhat Important)
+If an Event is Space Biome only and it needs you to have a Steel type with the moves Extreme Speed and Pluck and the ability Levitate because you need your Pokémon to fix a rocket on the fly, it might very well be ⚪ and still almost no one should find it. Because of this, **in some exceptional cases** you might see an Event with a catchable Mythical Pokémon in a 🟡 Ultra Event. This is because the Biome requirement itself and the other factors are making it much harder to appear. 
+
+__Thus, the more requirements an Event has to even proc, the more it can lean towards ⚪__
+
+### Simplicity (Somewhat Important)
+
+__The simpler the Event is at first glance **to the player**, the more it can lean towards ⚪ Common__.
+Pretty straightforward, and useful when you're dealing with doubts between ⚪ Common or 🔵 Great. 
+
+### What the Event rewards you with (**NOT** Important)
+"Wait, what?"
+
+__Excluding Meta Progression__ and __Swinginess__, I couldn't care less what the Event is handling you, it can very well be a Master-Ball __as long as the price you pay for it is right on par with it__. So go wild.
+
+### That's it for rarity/tier
+If you still can't figure it out, still suggest a range, like "I think either ⚪ Common or 🔵 Great".
+
+## What is "Waves"
+This is the floors/waves where you'd like your Event to happen. (11-179)* is the standard, but think about the following:
+
+- Is your Event too strong early, but balanced towards the later waves of Classic? Then you might want to consider going for something like 50-179, or 110-179.
+- Is your Event simply useless towards later stages? Then something like 11-89 might be your best call.
+- Do you love number four? You can set your Events to be only multiples of 4. Hey, if it makes sense... 🤷🏻‍♂️
+
+## Biomes
+
+Currently, you can set the Events to happen in the specific Biomes of your choosing, but there are other options too that you can use:
+- "ANY" ; self explanatory, it's any/all biomes
+
+- "EXTREME_BIOMES" and "NON-EXTREME_BIOMES" ; these are all the biomes in two mutually exclusive lists. The main point of this split is to avoid "Breeders in Space" type situations, with random trainers saluting you while on the moon.
```

Comment:
```suggestion
### `EXTREME_BIOMES` and `NON-EXTREME_BIOMES` 

these are all the biomes in two mutually exclusive lists. The main point of this split is to avoid "Breeders in Space" type situations, with random trainers saluting you while on the moon.
```